### ISSUE TITLE
Mutliple similar maneuvering charts

### ISSUE THREAD

**Issue by luc-girod** [raithos/xwing#317]
Hei!

When printing out with maneuvering charts, it prints out the charts for every ship, even if there's duplicates. Could a test be implemented to only print a chart once? For swarms, it really ends up taking a lot of space for repeat information...

**Comment by Patschke** [comment]
That can, in general, be implemented. I am however not sure, how exactly I'd want that to work: Where would we want to display that chart once? Should it be included with the first ship of that type, or would it be easier to give a list of all charts relevant for that squad somewhere at the end of the print out? How are we going to handle maneuver charts, that are not equal (e.g R4-Astromech to some of the ships)? Should we add another checkbox to the print preview thingy (there are already quite a few of them, and it starts getting messy), or would we enforce that behavior permanently?

**Comment by luc-girod** [comment]
The most elegant would probably be as a section head of a group of identical ships (if the user puts the ships together in the build, if the ships are not contiguous, then they all get their dial).

Or it could be to have all the charts at the bottom with an indication of what ship it's for ("T-70", "T-70+R4" and such). That is still better than having to pick your dial back up, mess it up a bit and lend it to your opponent for them to check it out ^^

**Comment by Patschke** [comment]
Both options are possible: Having all maneuver charts at the bottom of the squad list (once per ship), or having a chart at the first ship of that type. I wonder which way should be preferred.

**Comment by LordVorKon** [comment]
Personally, I think having all the charts in one place would be more useful overall.

**Comment by raithos** [comment]
I was never really pleased with the position of the charts on the ship itself to begin with. I would be for stripping them from the ship and adding a list of charts for each type of ship at the bottom of your list.

**Comment by Patschke** [comment]
Implemented in #324 

As always I know how I want things to work, but I am not at all sure how I want things to look like. I think it is a good idea, to allow multiple maneuver dials beside each other, to save space. Do we want to have some sort of frame around the dials section? Do we want to have a heading "Maneuver Dials"? Is it usefull to enforce, that all maneuver charts are printed on the same page? Do we want to align them on the left or right? 

![grafik](https://user-images.githubusercontent.com/17853362/54077781-a8d08200-42bd-11e9-836d-86d23ef7bc30.png)

**Comment by LordVorKon** [comment]
My opinion: 
All on the same/separate page: yes
Border/frame: no
Alignment: center
Heading: List name

**Comment by Patschke** [comment]
> 
> 
> My opinion:
> All on the same/separate page: yes

If they fit to the same page, I think there is no need to enforce creating a separate page. So my question was: If there is not enough space below the squad, should the available space be used and the remaining charts be put on the next page, or should all charts be placed on the next page. 

> Border/frame: no
> Alignment: center
> Heading: List name

I don't think I want the List name at that spot again...? Given above settings, it looks like that: 

![grafik](https://user-images.githubusercontent.com/17853362/54078236-1b912b80-42c5-11e9-8653-915d486cd79a.png)

Example with not enough space remaining: 

![grafik](https://user-images.githubusercontent.com/17853362/54078334-b8a09400-42c6-11e9-88a3-7348743c1296.png)


Relevant CSS:
```  .print-dials-container
    overflow: auto 
    text-align: center
    page-break-inside: avoid

    .print-dials-header
      text-align: left

    .fancy-dial
      padding: 5px
      table-layout: fixed
      display: inline-block```

**Comment by LordVorKon** [comment]
If it all fits then no a separate page probably isn't necessary.  Might be nice to have the option, however.

I do feel that all the charts ought to be on the same page, even if that means forcing a new page.

If it does go to a separate page then I think having the squad name repeated is handy, to ensure I know at a glance which list it goes with.

Given that my primary use of the maneuver charts will be to have it available to show my opponent, it would be awkward to have the wrong one.  Also, this is the reason for having all of them on the same page, for convenience.

**Comment by raithos** [comment]
I fixed a minor font color issue when printing for the . I am satisfied on how it looks now, I do not think its necessary to create a new page for keeping all the dials together in the case they get split. as well i don't think continuing share the list name is necessary. there should be an expectation that the player can keep his own papers in order. since the individual dial charts are labelled that should be all that is required. I'm going to close this as a result, the issue has been resolved
